app/views/idv/doc_auth/agreement.html.erb

```diff
@@ -19,18 +21,25 @@
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-  <%= check_box_tag(
-        :ial2_consent_given,
-        true,
-        false,
-        class: 'usa-checkbox__input',
-      ) %>
-  <label class="usa-checkbox__label margin-bottom-4" for="ial2_consent_given">
-    <%= t('doc_auth.instructions.consent') %>
-    <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
-  </label>
+  <div class="margin-bottom-4">
+    <%= check_box_tag(
+          :ial2_consent_given,
+          true,
+          false,
+          class: 'usa-checkbox__input',
+          required: true,
+          aria: {invalid: false}
+        ) %>
+    <label class="usa-checkbox__label" for="ial2_consent_given">
+      <%= t('doc_auth.instructions.consent') %>
+      <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
+    </label>
+    <div class="usa-error-message usa-error-message--with-icon display-if-invalid" role="alert">
+      <%= t('sign_up.agree_to_rules_of_use_and_continue') %>
```

Comment:
Per the [previous comment](https://github.com/18F/identity-idp/pull/5417#discussion_r7121333380), can we move / rename this string so it's not referencing "sign up" and "rules of use" if we intend to use it wherever there's a required checkbox to be checked? Maybe something like `forms.validation.required_checkbox` ?